src/MainApp.js

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from "react";
+import Main from "./components/Main";
+import Sidebar from "./components/Sidebar";
+import Navi from "./components/Navi";
+import NoteDisplay from "./components/NoteDisplay/";
+import { ThemeContext } from "./contexts/ThemeContext";
+import NoteInfo from "./components/NoteInfo";
+import { AppContext } from "./contexts/AppContext";
+
+export default function MainApp() {
+    const { darkMode } = useContext(ThemeContext);
+    const { state } = useContext(AppContext);
+
+    return (
+        <React.Fragment>
+            <Main
+                darkMode={darkMode}
+                showInfo={state.infoBarOpen}
```

Comment:
For boolean variables, I think a question like variable name is better? Kinda like `isInfoBarOpen` or `hasDrawer` kinda like that. It also helps signify that this variable returns a boolean result.